server/test/src/main/antlr4/org/eclipse/usecase/UseCasePreprocessor.g4

```diff
@@ -123,7 +129,8 @@ replacement
    ;
 
 identifier
-   : (IDENTIFIER | NUMBERLITERAL | LINKAGE | SECTION | DOT | INOF |STRINGLITERAL | TEXT)+
+   : (IDENTIFIER | NUMBERLITERAL | LINKAGE | SECTION | DOT | INOF | STRINGLITERAL | PROCEDURE_DIVISION
+     | END_DECLARATIVES | TEXT)+
```

Comment:
I don't think the new tokens should be included here given there is a white space in them.

Reply:
There are issues in functionDefinition if  PROCEDURE_DIVISION and I guess potentially END_DECLARATIVES are not in identifiers.